test/metalKey.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+import dom from 'metal-dom';
+import * as KEYMAP from '../src/keyConstants';
+import Component from 'metal-component';
+import IncrementalDomRenderer from 'metal-incremental-dom';
```

Comment:
Seems like `metal-incremental-dom` is missing as a dev dependency in package.json, I get an error when I install node_modules from scratch and try to run the test because of this.